python/cac_tripplanner/cms/models.py

```diff
@@ -1,9 +1,14 @@
+from ckeditor.fields import RichTextField
```

Comment:
I was reviewing the coding guidelines yesterday and tried to match it to that: "Imports should be grouped into three sections at the top of the file: Python built-in modules, third-party modules, and local source code (stuff in source control)." Maybe that needs to be revised to have a fourth section for django modules? In the meantime I'll switch it as you suggested.